back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -13,6 +13,16 @@ def index
       .order(:ordering)
     @publications = paginate @publications
 
+    if params[:search].present?
+      LogActivityJob.perform_later(
+        current_user,
+        'searched_admin_publications',
+        current_user,
+        Time.now.to_i,
+        payload: { search_query: params[:search] }
+      )
+    end
```

Comment:
👍

Reply:
Where does this show up?

Reply:
I actually forget 😆 But if you look at the `LogActivityJob` class, it is writing `Activity` records, so at the very least we can query these via Metabase. I think we do send them somewhere else - you can see Rabbit MQ stuff in that class, too:
```
def publish_activity_to_rabbit
  PublishActivityToRabbitJob.perform_later(activity)
end
```
 Maybe also to Matomo? (but I haven't really used Matomo much).

@alexander-cit probably knows more on this.

Reply:
That's a good question, I didn't think about it. They are definitely stored in the `activities` DB table but also are used somehow on the FE (look for `/activities` in `*.ts*` files.

Reply:
I believe Matomo stores only FE user interactions saved from the FE. 
And as Simon said, Metabase will eventually store activities too.